llvm/lib/IR/Constants.cpp

```diff
@@ -1384,6 +1440,16 @@ Constant *ConstantVector::getImpl(ArrayRef<Constant*> V) {
 
 Constant *ConstantVector::getSplat(ElementCount EC, Constant *V) {
   if (!EC.isScalable()) {
+    // Maintain special handling of zero.
```

Comment:
Just wondering whether this is something you want to keep long term or just initially?

Reply:
I'm happy to take offers but see it as temporary.